style.css

```diff
@@ -95,3 +121,91 @@ button:focus-visible {
     background-color: #f9f9f9;
   }
 }
+
+.modal {
+    display: none;
+    /*overflow: auto;*/
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0, 0, 0, 0.5);
+}
+
+.modal_content {
+    text-align: left;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-30%, 0%);
+    width: 30%;
+    border-radius: 10px;
+    background-color: white;
+    padding: 10px;
+}
+
+.modal_buttons {
+    float: right;
+}
+
+.OK_button {
+    background-color: royalblue;
+    color: white;
+
+}
+
+.OK_button:hover {
+    /*  I am not able to make color beautiful ((((  */
+    background-color: #4186e1;
+    color: white;
+}
+
+.modal_header {
+    padding-top: 1px;
+}
+
+.close_modal {
+    /* I don't know why it's not square form */
```

Comment:
You can ass styles like that:

```css
    /* to make it square form */
    height: 32px;
    width: 32px;

    /* to set content to the center */
    display: flex;
    align-items: center;
    justify-content: center;

    /* to make compensation for "x", because it is not really symmetrical*/
    padding-bottom: 4px;
```